test/benchmark/measure_performace.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
shebang removed.

The `benchmark` folder and its content is not specific to `rospack` at all so I can remove the commit that added it if that's better.